manager/dispatcher/dispatcher.go

```diff
@@ -214,11 +233,11 @@ func (d *Dispatcher) Run(ctx context.Context) error {
 		select {
 		case <-publishTicker.C:
 			publishManagers()
-		case <-d.processTaskUpdatesTrigger:
-			d.processTaskUpdates()
+		case <-d.processUpdatesTrigger:
+			d.processTaskAndNodeUpdates()
```

Comment:
How about changing `processTaskAndNodeUpdates` to `processUpdates`, like the change to trigger?